### PR TITLE
Fixes #77

### LINKED ISSUE #77: Implement scrollspy
According to design of the website, an underline will be much better than darkening the navbar item.

### PR DESCRIPTION
**Pull request by Archies11**
# Description

Added scrollspy and changed hover effect to underline in navbar.

Fixes #77

## Type of change

- [] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Live Gh-Pages Link
[Live Link] https://focused-snyder-33ef50.netlify.com/

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by Anupam-dagar — CHANGES_REQUESTED** [review]
I can't see the scrollspy happening, also change the underline to some good looking one, according to website color scheme, a purple one sounds good to me or a gradient one according to landing page. Also maybe try to underline not completely.
Also give a meaningful title to your PR. Only fixes isn't any good. You should write fixes in the description not in the title.

File: index.html [hunk 1]
```diff
@@ -20,10 +20,17 @@
   <link href="static/css/bootstrap.min.css" rel="stylesheet">
   <link href="static/css/style.css" rel="stylesheet">
   <script src="static/js/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js"></script>
+  <style>
+  body {
+      position: relative; 
+  }
```
Comment (Anupam-dagar) [inline index.html]: are you completely sure of this?

**Review by Anupam-dagar — CHANGES_REQUESTED** [review]
left some comments

**Comment by Anupam-dagar** [comment]
@Archies11 updates?

**Comment by Archies11** [comment]
@Anupam-dagar I tried to add scrollspy by following the instructions on different blogs and websites for bootstrap version used in the code but somehow I am unable to activate the feature. Is there any link that might help me with the same?

**Comment by Anupam-dagar** [comment]
This might help https://getbootstrap.com/docs/4.0/components/scrollspy/

**Comment by Archies11** [comment]
> This might help https://getbootstrap.com/docs/4.0/components/scrollspy/

I did read this Blog and implement the same but it didnt work :(

**Comment by Anupam-dagar** [comment]
@Archies11 check on codepen.

**Pull request closed without merging**